<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintEdit</class>
 <widget class="QWidget" name="PrintEdit">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/image/logo2.PNG</normaloff>:/image/logo2.PNG</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(244, 244, 244);
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background-color: rgb(116, 126, 146);
	border-radius: 12px;
}

/*QComboBox::drop-down
{
	
	subcontrol-origin: margin;
}*/
QLineEdit
{	
	color: rgb(255, 255, 255);
	border-radius: 4px;
	border: 1px solid rgb(255,250,250);
	background-color: rgb(116, 126, 146);
}
QLabel
{
	font-family:&quot;Source Sans Pro&quot;;
	font-size: 12px;
    font-weight: regular;
	color:#FFF;
}

QPushButton,QToolButton
{
   /* color: #424344;*/
    font-family: &quot;Source Sans Pro&quot;;
    font-size: 11px;
    font-weight: regular;
 	border: 1px solid yellow;
 	border-radius: 4px;
    padding: 0px 15px;   
    max-height: 100px;
    min-height: 25px;
	min-weight:52px;

 	/*background-color: #2288C1;*/
	background-color: rgb(155,155,155);
    border-color: rgb(155,155,155);
    color: #fff;
}

QToolButton:pressed, QPushButton:pressed, QPushButton[btn-secondary=true]:pressed, QPushButton[btn-icon=true]:pressed
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #E0E1E3, stop:.25 #F6F7F9);
}

QPushButton:disabled
{
    background-color: #EAEBED;
    border-color: #E1E2E4;
    color: #C0C1C3;
}

QPushButton:pressed,QToolButton:pressed
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #0072a9, stop:.25 #2288C1);
}
QPushButton:hover,QToolButton:hover
{
    background-color: #6C22E2;
}

QRadioButton{
	color:#fff;
}
QGroupBox
{
  border: 1px solid #ffffff;
  border-radius:3px;
  color: #FFF;
  background-color: rgb(116, 126, 146);

}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextBrowser" name="InforBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser
{
	color: rgb(255, 255, 255);	
	background-color: rgb(115, 115, 115);
    font-family: &quot;Source Sans Pro&quot;;
    font-size: 14px;
    font-weight: regular;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Source Sans Pro'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Clean the infor browser</string>
    </property>
    <property name="text">
     <string>清屏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrintButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>143</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Begin print</string>
    </property>
    <property name="text">
     <string>打印</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopPrintButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>144</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>stop print,can recover again</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecoverButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>177</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Go on print</string>
    </property>
    <property name="text">
     <string>恢复</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>176</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Reset the printer.</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="IPGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>端口连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <widget class="QPushButton" name="ConnectButton">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>50</y>
       <width>52</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Connecte to the printer</string>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QLabel" name="IP_label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>打印前，请连接打印机，或者选择脱机打印</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>空</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="ImageButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>110</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Show the layer image</string>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SliceButton">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>110</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Slice the object</string>
    </property>
    <property name="text">
     <string>切片</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
